### ISSUE TITLE
China Union Pay Issue

### ISSUE THREAD

**Issue by QuintonC** [jessepollak/card#417]
It looks the dist/release is missing the `jp-card-unionpay` class within the `Card.prototype.cardTemplate` init.

See here: (Note that I actually broke this down and made it readable... also I adjusted some of the strings in here (jp-card-elo, jp-card-dankort, jp-card-jcb) so that they were in-line like the others.

```
Card.prototype.cardTemplate = '' + 
	'<div class="jp-card-container">' + 

		'<div class="jp-card">' + 

			'<div class="jp-card-front">' + 

				'<div class="jp-card-logo jp-card-elo"><div class="e">e</div><div class="l">l</div><div class="o">o</div></div>' + 

				'<div class="jp-card-logo jp-card-visa">Visa</div>' + 
				
				'<div class="jp-card-logo jp-card-visaelectron">Visa<div class="elec">Electron</div></div>' + 
				
				'<div class="jp-card-logo jp-card-mastercard">Mastercard</div>' + 

				'<div class="jp-card-logo jp-card-maestro">Maestro</div>' + 

				'<div class="jp-card-logo jp-card-amex"></div>' + 
				
				'<div class="jp-card-logo jp-card-discover">discover</div>' + 
				
				'<div class="jp-card-logo jp-card-dinersclub"></div>' + 

				'<div class="jp-card-logo jp-card-dankort"><div class="dk"><div class="d"></div><div class="k"></div></div></div>' + 
				
				'<div class="jp-card-logo jp-card-jcb"><div class="j">J</div><div class="c">C</div><div class="b">B</div></div>' + 

				'<div class="jp-card-lower">' + 
					'<div class="jp-card-shiny"></div>' + 
					'<div class="jp-card-cvc jp-card-display">{{cvc}}</div>' + 
					'<div class="jp-card-number jp-card-display">{{number}}</div>' + 
					'<div class="jp-card-name jp-card-display">{{name}}</div>' + 
					'<div class="jp-card-expiry jp-card-display" data-before="{{monthYear}}" data-after="{{validDate}}">{{expiry}}</div>' + 
				'</div>' + 

			'</div>' + 
				
			'<div class="jp-card-back">' + 
				'<div class="jp-card-bar"></div>' + 
				'<div class="jp-card-cvc jp-card-display">{{cvc}}</div>' + 
				'<div class="jp-card-shiny"></div>' + 
			'</div>' + 
			
		'</div>' + 

	'</div>';
```

@jessepollak is this a known issue or..? I looked back at some previous issues, specifically #99 and #192 but it doesn't look like Union Pay ever made it into the system fully.

Should I proceed by opening a PR?

**Comment by kimchakabuto** [comment]
Cash

**Comment by kimchakabuto** [comment]
Cash

**Comment by Phuchong1** [comment]
Cash

**Comment by melloware** [comment]
Fixed in 2.5.0
